### DISCUSSION TITLE
Prompt burn-in?

### DISCUSSION THREAD

**Discussion by edwardsdigital** [comfyanonymous/ComfyUI#3856] [Q&A]
Im having a weird issue occasionally that, due to a lack of a better description, I will call it “prompt burn-in”. It’s almost like the system gets a prompt stuck in its head. If I do multiple back to back generations on a prompt with very minor changes when I’m trying to dial in a result, and then change the prompt to generate a totally different concept, the system will continue to generate images similar to the first prompt. Sometimes I have to close comfy for a few minutes to get it to essentially reset and allow generation of a new concept.

Has anyone else noticed this, and is there  fix for it besides what I’m already doing? If there’s already a thread relating to this I’d appreciate a link since my searches didn’t turn anything up and I haven’t gotten deep enough into how stable diffusion works to know why this happens yet to make a more informed search.

**Comment by edwardsdigital** [comment]
After a bit of searching online I found an old thread on reddit talking about prompt ghosting which sounds like what I have experienced.

[https://www.reddit.com/r/StableDiffusion/comments/12ub9ps/is_prompt_ghosting_a_thing_old_prompts/](url)

They talk about it being an xformers artifact so I am going to attempt a few experiments and see what I can figure out.

**Reply by muhammadyusuf-kurbonov** [reply]
And what have you found?

**Comment by RubenTainoAI** [comment]
What values are you using for seed. If the seed is the same, and the prompt kind of similar, the results is the same or similar video. Use Random or increasing seeds. If you need to keep closing comfy, it's faster to just refresh or restart it, using the manager.
